src/app/view-config-versions.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable camelcase */
+import Ajv from 'ajv';
+import configSchema0_1_0 from '../schemas/config-0.1.0.schema.json';
+import configSchema1_0_0 from '../schemas/config-1.0.0.schema.json';
+import configSchema1_0_1 from '../schemas/config-1.0.1.schema.json';
+import configSchema1_0_2 from '../schemas/config-1.0.2.schema.json';
+
+import cellSetsSchema from '../schemas/cell-sets.schema.json';
+import rasterSchema from '../schemas/raster.schema.json';
+import {
+  upgradeFrom0_1_0,
+  upgradeFrom1_0_0,
+  upgradeFrom1_0_1,
+} from './view-config-upgraders';
+
+/**
+ * Mapping from view config versions to view config schemas and upgrade functions.
+ * Add a new schema and upgrade function here when bumping the view config version.
+ * The latest view config version should always have a null value instead of an upgrade function.
+ */
+export const LATEST_VERSION = '1.0.2';
+export const UPGRADE_FUNCTIONS = {
```

Comment:
Now sure what else to call this, but the last function here never "upgrades" really.  Maybe `SCHEMA_HANDLERS`?  Not sure that's better since it goes from conveying not enough to not specifying much of anything....guess we can just leave it.